Makefile

```diff
@@ -89,6 +88,9 @@ lib-dep:
 		-r requirements-release.txt \
 		-r requirements-tunnel.txt
 
+lib-scm-version:
+	@echo "version = '$(VERSION)'" > proxy/common/_scm_version.py
```

Comment:
This misses out on the `tuple_version`

Reply:
Can you expand, per `_scm_version.pyi`

```
# This stub file is necessary because `_scm_version.py`
# autogenerated on build and absent on mypy checks time
version: str
```

So I simply dumped this version string in there.  It seems to work.  Are we missing something here?